.jenkins/tests/pom.xml

```
@@ -299,9 +299,9 @@
                 </includes>
             </testResource>
             <testResource>
-                <directory>${project.basedir}/../../dsl/seed</directory>
+                <directory>${project.basedir}/../../dsl/seed/jobs</directory>
                 <includes>
-                    <include>Jenkinsfile.seed</include>
+                    <include>Jenkinsfile.*</include>
```

```suggestion
                    <include>Jenkinsfile*</include>
```

may be better :thinking:

---

gonna update

---

updated